src/actors/rqgActor.ts

```diff
@@ -300,4 +300,9 @@ export class RqgActor extends Actor {
       }
     }
   }
+
+  public getItemsByRqid(rqid: string): RqgItem[] {
```

Comment:
changed the name to getEmbeddedItemsByRqid.

Put "Mostali" in the Mostali template's species (instead of human).  I don't know how many times I fixed that and lost other changes to the templates while editing this.  I think we need to get the compendia for the base game to generate from yaml too so we don't loose changes.

Fixed the runes values, which was messed up because it was putting a function into the "chance" rather than the value returned by the function.